src/actions/actions.ts

```diff
@@ -2112,8 +2112,8 @@ class CommandGoToDefinition extends BaseCommand {
 
 @RegisterAction
 class CommandInsertAtFirstCharacter extends BaseCommand {
-  modes = [ModeName.Normal];
-  mustBeFirstKey = true;
+  modes = [ModeName.Normal, ModeName.Visual];
+  mustBeFirstKey = false;
```

Comment:
I *think* the reason was (in the case of i) so that `ciw` would not cause an Insert to trigger and make things get really confused.

I'm not sure if that applies here, though! I'm not sure if `I` is inside of another Vim command somewhere... I don't think it is, but then again, it's a bit scary...